README.md

```diff
@@ -97,6 +97,24 @@ BRANCHOUT_NAME=shorter-name
 
 to the Branchoutfile and ideally use the same value as the optionalDirName during branchout init.
 
+### Personal customisation
+
+Sometimes the branchout defaults don't work for a particular scenario even though they do work for most of your colleagues.
+One example is that the `${HOME}` places the two root directories on a very slow network drive.
+Another might be that you already have a tree with your code and you want to keep it all together.
+In cases like these you can use the following two files to ????
+
+```
+~/.config/Branchoutrc?
+${BRANCHOUT_BASE}/.branchoutrc
+```
+#### Moving the branchout projects base dir
+
+To set the root of where branchout initialises branchout repos do XXX
```

Comment:
set BRANCHOUT_PROJECTS_DIRECTORY in ~/.config/branchoutrc

Reply:
@stickycode Is this true? Where I saw this used it was used on top of `${HOME}` and it would be perfectly reasonable to put it on another drive mounted under for example `/code` or whatever that's outside `~/` - one such reason that I've hit in the past is limited path names in ubuntu with encrypted home functionality. I had to have my repos outside of ~/ for that reason. 

Do we need a new variable that overrides both this and the base path being $HOME or just a wrapper layer around home that uses home as a default and lets you override it to eg / or wherever else you want it?